pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -127,98 +166,93 @@ def clear(self):
+        return True
+
+    def _get_long_identifier(self, disk):
         # For iSCSI devices, we want the long ip-address:port-iscsi-tgtname-lun-XX
         # identifier, but blivet doesn't expose that in any useful way and I don't
         # want to go asking udev.  Instead, we dig around in the deviceLinks and
         # default to the name if we can't figure anything else out.
         for link in disk.device_links:
             if "by-path" in link:
-                lastSlash = link.rindex("/")+1
-                return link[lastSlash:]
+                last_slash = link.rindex("/") + 1
+                return link[last_slash:]
 
         return disk.name
 
+
 class SearchPage(FilterPage):
     # Match these to searchTypeCombo ids in glade
-    SEARCH_TYPE_NONE = 'None'
     SEARCH_TYPE_PORT_TARGET_LUN = 'PTL'
     SEARCH_TYPE_WWID = 'WWID'
 
     def __init__(self, storage, builder):
-        super().__init__(storage, builder)
-        self.model = self.builder.get_object("searchModel")
-        self.model.set_visible_func(self.visible_func)
-
-        self._lunEntry = self.builder.get_object("searchLUNEntry")
-        self._wwidEntry = self.builder.get_object("searchWWIDEntry")
-
-        self._combo = self.builder.get_object("searchTypeCombo")
-        self._portCombo = self.builder.get_object("searchPortCombo")
-        self._targetEntry = self.builder.get_object("searchTargetEntry")
-
-    def setup(self, store, selectedNames, disks):
-        self._combo.set_active_id(self.SEARCH_TYPE_NONE)
-        self._combo.emit("changed")
+        super().__init__(storage, builder, "searchModel", "searchTypeCombo")
+        self._lun_entry = self._builder.get_object("searchLUNEntry")
+        self._wwid_entry = self._builder.get_object("searchWWIDEntry")
+        self._port_combo = self._builder.get_object("searchPortCombo")
+        self._target_entry = self._builder.get_object("searchTargetEntry")
 
+    def setup(self, store, selected_names, disks):
         ports = []
+
         for disk in disks:
             if hasattr(disk, "port") and disk.port is not None:
                 ports.append(str(disk.port))
 
-        self.setupCombo(self._portCombo, ports)
+        self._setup_combo(self._port_combo, ports)
+        self._setup_search_type()
 
     def clear(self):
-        self._lunEntry.set_text("")
-        self._portCombo.set_active(0)
-        self._targetEntry.set_text("")
-        self._wwidEntry.set_text("")
+        self._lun_entry.set_text("")
+        self._port_combo.set_active(0)
+        self._target_entry.set_text("")
+        self._wwid_entry.set_text("")
 
     def _port_equal(self, device):
-        active = self._portCombo.get_active_text()
+        active = self._port_combo.get_active_text()
         if active:
             if hasattr(device, "port"):
                 return device.port == int(active)
-            else:
-                return False
-        else:
-            return True
+
```

Comment:
Please add else here, it will be easier to follow the
```
    if <condition>:
        return
    return
return
```

```suggestion
        else:
```

Reply:
Fixed.